src/test/datascience/kernel-launcher/localKernelFinder.unit.test.ts

```diff
@@ -1129,15 +1134,24 @@ import { JupyterServerUriStorage } from '../../../kernels/jupyter/launcher/serve
                                 },
                                 {
                                     interpreter: python2Global
+                                },
+                                {
+                                    interpreter: python38VenvEnv,
+                                    kernelSpecs: [fullyQualifiedPythonKernelSpec]
                                 }
-                                // },
-                                // {
-                                // interpreter: python38VenvEnv,
-                                // kernelSpecs: [fullyQualifiedPythonKernelSpec]
-                                // }
```

Comment:
Not sure why this was disabled, enabling this.
The more intepreters we have in our test, the better. yes it complicates the tests.
But having more interpreters is a valid scenario.